grammar.pegjs

```diff
@@ -0,0 +1,111 @@
+document = document_title events:event_month
+{
+  return events
+}
+
+document_title = "#" text newline newline (text newline)+ newline (text newline)+ newline newline
```

Comment:
I think we should not parse the document header section, as it is subject to change.

This pattern seems to only allow two paragraphs of heading, and if I want to add more, I have to edit the grammar. This can make it hard to maintain.

I would break the whole document into sections, and parse them separately.